### DISCUSSION TITLE
Remove ring of floating carpets around player?

### DISCUSSION THREAD

**Discussion by erikeckhardt** [gnembon/fabric-carpet#1430] [General]
For the life of me, I can't figure out how to turn off the ring of floating carpets around the player. They're super distracting and annoying. Help?

**Comment by altrisi** [comment]
You should be able delete it by running `/script remove carpets`, or by going to your world's `scripts` folder and removing `carpets.sc`.

Why did you install the floating carpets script in the first place? (or how did it happen, for some reason this isn't the first time and I'd like to know what's going on)

**Reply by erikeckhardt** [reply]
I may have followed instructions somewhere online, believing that I needed to install the carpets script in order to gain access to the functionality of the carpet mod. What does the carpets script do if not give access to carpet commands?

Reading the README doesn't really communicate enough context to get me going. I'm a software developer and like to think I'm not particularly stupid, so maybe some additional context would be useful about what the parts of the carpet mod are, how to use them, how they display, etc.

If the carpets always have to float around when the script is on, why? Do I need them? Can we change it so they don't display? Or do they do something? I couldn't find anything anywhere indicating this information.

Maybe I'll modify the source code, but a quick scan has so far not revealed where the code is that causes the floating carpet ring...
